app/assets/stylesheets/report.scss

```diff
@@ -32,3 +45,13 @@
     text-align: center;
   }
 }
+
+.rlabel {
+  display: inline-block;
+  text-align: right;
+  margin-right: 5px;
+  padding-right: 5px;
+  border-right: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.